pm_utils/business.py

```diff
@@ -0,0 +1,115 @@
+"""
+Add modules that are shared in other scripts.
+"""
+
+# First come standard libraries, in alphabetical order
+import csv
+from collections import defaultdict
+import logging
+import re
+
+# After a blank line, import third-party libraries
+import pandas as pd
+
+# After another blank line, import local libraries
+from .utils import normalize_name
+from .rpt_columns import rpt_merge_cols
+from .mappings import STUDY_MAPPING
+
+logger = logging.getLogger("business in pm_utils")
```

Comment:
Using `__name__` is usually the standard

Reply:
```python
logger = logging.getLogger(__main__)
```

Reply:
Should really use `__name__` instead of `__main__`.
Will fix this in another branch.